cerberus/cerberus.py

```diff
@@ -454,6 +454,7 @@ def __normalize_mapping(self, mapping, schema):
         if self.purge_unknown:
             self._normalize_purge_unknown(mapping, schema)
         self._normalize_coerce(mapping, schema)
+        self._normalize_default(mapping, schema)
```

Comment:
though i have only one silly argument for setting defaults before coercing, here it is: it seems more intuitive to normalize in that order: purge unwanted items, fill up missing items, work on items.

but maybe it's be useful to coerce a value to `None` in order to replace it with a default. coercing defaults on the other hand shouldn't make sense.

so propably it's better to keep it like it is. but i'd be interested in other thoughts.